indico/modules/events/contributions/controllers.py

```diff
@@ -113,3 +111,22 @@ def _process(self):
         flash(ngettext("Contribution was deleted.",
                        "{count} contributions were deleted.", deleted_count).format(count=deleted_count), 'success')
         return jsonify_data(html=_render_contribution_list(self.event_new))
+
+
+class RHContributionREST(RHManageContributionBase):
+    def _process_DELETE(self):
+        delete_contribution(self.contrib)
+        flash(_("Contribution '{}' successfully deleted").format(self.contrib), 'success')
```

Comment:
Yeah, I noticed it at some point and I have a commit that fixes them.
https://github.com/nop33/indico/commit/3c0b8d6e9f20e330c40b9a081481e772edb8399e